### DISCUSSION TITLE
Support of wmv3 codec

### DISCUSSION THREAD

**Discussion by Glebsin** [ppy/osu#27998] [Q&A] [answered]
Seems lazer missing wmv3 codec, video is not playing on this map (.avi, Windows Media Video 9, wmv3 main)  https://osu.ppy.sh/b/1476582

**Comment by Joehuu** [comment]
cc @FreezyLemon

**Reply by FreezyLemon** [reply, marked as answer]
osu-framework should enable `vc1` and `wmv3` decoder support (they're basically the same format, vc1 comes for free with wmv3). There's also hwaccel support for the formats.

I'll get around to adding that after the [upgrade to 7.0](https://github.com/ppy/osu-framework/pull/6256) is done.

**Reply by Glebsin** [reply]
Is it fixed now ?

**Reply by tadatomix** [reply]
It doesn't. See: https://github.com/ppy/osu-framework/issues/6625
